torch/distributed/_tensor/placement_types.py

```diff
@@ -381,16 +381,25 @@ def __hash__(self) -> int:
         # Caveat: we need to keep this in mind and sync hash and eq if we add more
         # fields to them,
         if self.tensor_meta is not None:
-            return hash((self.mesh, tuple(self.placements), self.tensor_meta.shape))
+            return hash(((self.mesh, self.placements), self.tensor_meta.shape))
```

Comment:
nit: can hash three together instead of two tuples
```
hash((self.mesh, self.placements, self.tensor_meta.shape))
```

Reply:
sure, I was about to ask why we do something like this.